src/hook/useDarkMode.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react';
+
+const useDarkMode = () => {
+  const [theme, setTheme] = useState('light');
+  const colorTheme = theme === 'dark' ? 'light' : 'dark';
```

Comment:
Does this work as intended? The app is unable to detect my default browser theme.

Reply:
They way it is currently working, the default theme is light. The user will switch to the dark mode when they want to.

Reply:
Part of our discussion was that the app should be able to detect user theme and present light or dark theme accordingly.

Reply:
Let's look into that

Reply:
Let me know when this is ready, also remember to change the title.

Reply:
We suggest that we add that in the fixes that we have to work on, since this PR is currently blocking the rest.

Reply:
Alright, go ahead and merge it.